builtin/logical/transit/path_encrypt.go

```diff
@@ -482,33 +488,35 @@ func (b *backend) pathEncryptWrite(ctx context.Context, req *logical.Request, d
 			warnAboutNonceUsage = true
 		}
 
-		var factory interface{}
+		factories := make([]any, 0)
+		if ps, ok := d.GetOk("padding_scheme"); ok {
```

Comment:
Should this key not be a batch item parameter for both encode/decode operations, instead of parsing the value batch wide?